microsite/data/plugins/analytics-module-generic.yaml

```diff
@@ -0,0 +1,9 @@
+---
+title: 'Analytics Module: Generic'
```

Comment:
I'm thinking we might want to add HTTP in the title just to make it a bit clearer what transport this plugin uses.
```suggestion
title: 'Analytics Module: Generic HTTP'
```